FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -554,9 +554,14 @@ - (void)retrievalProcess:(id<FIRDLRetrievalProcessProtocol>)retrievalProcess
   self.retrievingPendingDynamicLink = NO;
   _retrievalProcess = nil;
 
-  if (!result.error && ![_userDefaults boolForKey:kFIRDLOpenURLKey]) {
```

Comment:
Should all errors be disregarded here or just errors specific to ipv4/ipv6 not being found?

Reply:
It makes sense for all errors to be disregarded ; otherwise it will result in retry on next app start and if it returns a pending link during retry, it would be an unexpected deeplink for the user.
@dmandar let me know your thoughts if you think otherwise.

Reply:
Makes sense to me.